share/messages_to_error_log.txt

```diff
@@ -11696,6 +11696,10 @@ ER_XB_MSG_6
 ER_XB_UNDO_DELETE_FAILURE
   eng "Deletion of 5.7 undo tablespace %s failed"
 
+ER_XB_MSG_DOWNGRADING_LOG_FILE
```

Comment:
Please, maybe we should create the new error codes at the end of Percona's list.

It might not be important, but this way, we do not change the numeric code of the existing errors. Please, think that there might be scripts checking the error numeric code instead of the text representation.

Reply:
fixed.